### PR TITLE
feat(client, server): add db logs and refactor db server side

### PR DESCRIPTION
**Pull request by Akirtovskis**
Can be only merged after : https://github.com/ledokku/ledokku/pull/101 
as logs views from there are used.

Will resolve : 
https://github.com/ledokku/ledokku/issues/22
https://github.com/ledokku/ledokku/issues/21
https://github.com/ledokku/ledokku/issues/20

@pradel this pr adds logs for db view and also I realised that we can easily tweak db server side code to make all dbs work right away. 

It works because for most common db plugins commands are exactly the same. 

What do you think about this approach?

### CODE REVIEW

**Comment by Akirtovskis** [comment]
Screenshot 2020-07-20 at 18 25 36

**AND**

Screenshot 2020-07-20 at 18 33 00

**Comment by Akirtovskis** [comment]
@pradel a bit of a scroll, but for now will do the job, I think.

localhost_3000_database_d35141c3-5b82-4bb3-8224-833180e20f5f_logs (1)

File: server/src/graphql/queries/databaseInfo.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    throw new Error('Unauthorized');
+  }
+
+  const database = await prisma.database.findOne({
+    where: {
+      id: databaseId,
+    },
+  });
+
+  if (!database) {
```
Comment (pradel) [inline server/src/graphql/queries/databaseInfo.ts:22]: c

File: server/src/graphql/queries/databaseLogs.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    throw new Error('Unauthorized');
+  }
+
+  const database = await prisma.database.findOne({
+    where: {
+      id: databaseId,
+    },
+  });
+
+  if (!database) {
```
Comment (pradel) [inline server/src/graphql/queries/databaseLogs.ts:22]: We need to check that database belongs to the user

**Comment by Akirtovskis** [comment]
@pradel both feedbacks fixed!

**Pull request merged**